skyportal/handlers/api/internal/instrument_observation_params.py

```diff
@@ -0,0 +1,17 @@
+import json
+from baselayer.app.access import auth_or_token
+from ...base import BaseHandler
+
+
+class InstrumentObservationParamsHandler(BaseHandler):
+    @auth_or_token
+    def get(self):
+        try:
+            with open("instrument_observation_params.json") as f:
+                instrument_data = json.load(f)
+        except FileNotFoundError:
+            return self.error("instrument_observation_params.json does not exist.")
```

Comment:
This leaks implementation details to the user.  How about something like: "Instrument observation parameters unavailable."

Reply:
Sure, done.